app/uk/gov/hmrc/eusubsidycompliancefrontend/services/SubsidyJourney.scala

```diff
@@ -91,4 +93,16 @@ object SubsidyJourney {
     Json.format[FormPage[TraderRef]]
 
   implicit val format: Format[SubsidyJourney] = Json.format[SubsidyJourney]
+
+  def fromSubsidy(nonHmrcSubsidy: NonHmrcSubsidy): SubsidyJourney = {
```

Comment:
Might be better to call his `fromNonHmrcSubsidy` to reflect the argument type that it takes.